io.gitlab.gregorni.ASCIIImages.json

```diff
@@ -0,0 +1,86 @@
+{
+    "app-id" : "io.gitlab.gregorni.ASCIIImages",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "asciiimages",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.portal.OpenURI"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+    	{
+	    "name": "blueprint-compiler",
```

Comment:
there should be a cleanup rule to remove it all since it's only needed at build time.

Reply:
How would I do that?